src/modules/client/pages/ni/auxiliaries/Dashboard.vue

```diff
@@ -226,7 +233,7 @@ export default {
         for (const sector of sectorsIds) {
           if (this.auxiliariesStats[sector]) continue;
           sectors.push(sector);
-          this.$set(this.displayStats[sector], 'loadingDetails', true);
+          set(this.displayStats[sector], 'loadingDetails', true);
```

Comment:
`this.displayStats[sector].loadingDetails = true;` ne fonctionnerait pas ?

Reply:
on utilisait this.$set pour une question de reactiviité dans vue 🤔 je ne sais pas si c'est toujours le cas dans vue 3

Reply:
a supprimer